js/util/config.js

```diff
@@ -1,6 +1,16 @@
 'use strict';
+// @flow
 
-module.exports = {
+type Config = {|
```

Comment:
What's the function of the `|`s? I don't see this syntax used in the [Flow Object Types documentation](https://flowtype.org/docs/objects.html#_).